src/CloudKit/CKCompat.cs

```diff
@@ -26,14 +26,30 @@ public partial class CKQueryNotification {
 #if !XAMCORE_4_0 && !WATCH
 	public partial class CKOperation {
 
+#if !NET
 		[Obsoleted (PlatformName.iOS, 9,3, message: "Do not use; this API was removed and will always return 0.")]
+#else
+#if IOS
+		[Obsolete ("Starting with ios9.3 do not use; this API was removed and will always return 0.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+#endif
```

Comment:
This is a case where the version number is not helpful (but was required by `[Obsoleted]`) since the API will always return `0` whatever which OS version is executing.